### DISCUSSION TITLE
MILESTONE 2: PROFILE PAGES!!!

### DISCUSSION THREAD

**Discussion by EnZon3** [EnZon3/Drop#10] [Announcements]
The overall structure of the profile pages are done. Lmk if there are any issues!

**Comment by EnZon3** [comment]
Profile page example: https://drop-beta.vercel.app/@EnZon3

**Comment by MilesWK** [comment]
SWEET! GOOD WORK!

**Reply by EnZon3** [reply]
Thank you so much Miles! You've been a big motivation to continue this project!

**Reply by EnZon3** [reply]
Try changing ur bio in the settings btw!

**Comment by EnZon3** [comment]
Next milestone is projects.

**Comment by MilesWK** [comment]
This is all INSANE! GOOD GOOD WORK, LIAM!

**Reply by EnZon3** [reply]
thank you so much man!
